modules/installation-installer-provisioned-vsphere-config-yaml.adoc

```diff
@@ -42,7 +42,7 @@ networking:
     hostPrefix: 23
   machineNetwork:
   - cidr: 10.0.0.0/16
-  networkType: OVNKubernetes <9>
+  networkType: OVNKubernetes <11>
```

Comment:
In case a merge reviewer notices that this is out of order: this is out of scope and should be addressed as a separate installer docs cleanup effort and Shubha doesn't need to worry about it :slightly_smiling_face:

Reply:
Thank you for the thoughtful note @jeana-redhat . Since I have merge rights, I will take a note :smile: